core/src/tasks/appRefreshQuery/check.ts

```diff
@@ -7,28 +7,28 @@ export class AppRefreshQueryCheck extends CLSTask {
   constructor() {
     super();
     this.name = "appRefreshQuery:check";
-    this.description = "check all appRefreshQueries and run them if it is time";
-    this.frequency =
-      process.env.GROUPAROO_RUN_MODE === "cli:run" ? 0 : 1000 * 60 * 5; // Run every 5 minutes
+    this.description =
+      "run a single appRefreshQuery and update values/trigger schedules if needed";
+    this.frequency = 0;
     this.queue = "apps";
-    this.inputs = {};
+    this.inputs = { appRefreshQuery: { required: true } };
   }
 
-  async runWithinTransaction() {
-    const appRefreshQueries = await AppRefreshQuery.findAll();
+  async runWithinTransaction(params) {
```

Comment:
You can type/destructure params to make it clear that only IDs are getting passed in: `runWithinTransaction({appRefreshQueryId}: {appRefreshQueryId:string})`